src/gui/plugins/transform_control/TransformControl.cc

```diff
@@ -712,7 +712,8 @@ void TransformControlPrivate::HandleMouseEvents()
       {
         this->transformControl.Node()->SetUserData(
           "pause-update", static_cast<int>(1));
-      } catch (std::bad_variant_access &)
+      }
+      catch (std::bad_variant_access &)
```

Comment:
fixed codestyle and conveniently it's one of the cases where I suppose try-catch block is redundant

Whole list:
TransformControl.cc:479
TransformControl.cc:558
TransformControl.cc:592
TransformControl.cc:681
TransformControl.cc:694
TransformControl.cc:716

Reply:
I think it may not be necessary as well. There are other places in the code base that don't wrap `SetUserData` with try/catch

Reply:
Ok, I'll clean-up it